### PR TITLE
Changes for kb

### PR DESCRIPTION
**Pull request by ethanstrominger**
Fixes #replace_this_text_with_the_issue_number

### What changes did you make?

-
-
-

### Why did you make the changes (we will use this info to test)?

-
-
-

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>

image

</details>

### CODE REVIEW

File: app/core/api/serializers.py [hunk 1]
```diff
@@ -202,8 +202,8 @@ class ProgramAreaSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = ProgramArea
```
Comment (ethanstrominger) [inline app/core/api/serializers.py:204]: Go back to uuid

File: app/core/api/urls.py [hunk 2]
```diff
@@ -34,7 +38,11 @@
 )
 
 urlpatterns = [
-    path("me/", UserProfileAPIView.as_view(), name="my_profile"),
+    path('secure-api/getusers', SecureApiView.as_view(), name='secure_api_get_users'),
+    path('secure-api/newcognitouser', SecureCreateUserFromCognito.as_view(),name='secure_new_user'),
+    path('me/', UserProfileAPIView.as_view(), name='my_profile'),
+    path('home/', TemplateView.as_view(template_name='common/home.html'), name='home'),
```
Comment (ethanstrominger) [inline app/core/api/urls.py:44]: fix this one

**Review by fyliu — CHANGES_REQUESTED** [review]
Maybe it's too early for me to review this since several things aren't complete yet.

Looks like there's a lot of directions to the changes. Some of them might be better split into their own PRs to review separately.

1. cognito auth client in the backend
    - missing instructions on how to use it. What url to use. Needing to create the client id in AWS and put it in the .env file.
1. things not needed
    - changing model id between int and uuid (it's also causing existing tests to fail)
    - changing the .env file name
    - envsettings.sh file
1. shouldn't be needed
    - API for peopledepot clients to create new peopledepot users
    - default assigning staff to new users allows all users to use the admin site
1. scripts and instructions for running the backend without docker

File: app/templates/account/login.html
Comment (fyliu) [inline app/templates/account/login.html:1]: I think it's `accounts/login.html`. Accounts with an s at the end.

**Pull request closed without merging**